meta/meta-node/src/lib.rs

```diff
@@ -33,11 +33,16 @@ mod ffi {
     }
 
     extern "Rust" {
+        fn connect_block(payload: ffi::DmcBlock) -> Result<()>;
```

Comment:
Why is this ffi::DmcBlock? It's part of the same module.

Reply:
updated! thanks